packages/libs/error-stack/tests/snapshots/test_serialize__full__attachment.snap

```diff
@@ -0,0 +1,43 @@
+---
+source: tests/test_serialize.rs
+expression: report
+---
+[
+  {
+    "context": "root error",
+    "attachments": [
+      {
+        "file": "tests/common/mod.rs",
+        "line": 4,
+        "column": 5
+      },
+      "backtrace (1)",
```

Comment:
Can we/do we want to serialize backtraces as well?

Reply:
We could serialize them as a string. Returning the backtrace structurally is a bit more complicated, because we'd likely need to regex match or *somehow* convert/use the backtrace crate.

Reply:
I think, as long as `Serialize` is not implemented on `Backtrace` upstream, we should serialize it as string. If an upstream implementation is provided at some point we can switch to that.

Reply:
fixed in [efc3db4](https://github.com/hashintel/hash/pull/1558/commits/efc3db4212624c1564e5de5d024f6fca916a7ede)